### PR TITLE
Feature 15 survey properties edit draw

### PR DESCRIPTION
**Pull request by Jdallos**

### CODE REVIEW

File: src/components/screens/edit-surveys-screen.tsx [hunk 1]
```diff
@@ -32,23 +33,41 @@ const EditSurveysScreen = () => {
...
+  const onSaveSurvey = async ({ target: { value, name } }: ChangeEvent<HTMLInputElement>) => {
+    if(!survey.id) return;
+
+    const editedSurvey = {
+      ...survey,
+      [name]: value
+    };
+    const updatedSurvey = await surveysApi.updateSurvey({ surveyId: survey.id, survey: editedSurvey });
+
+    setSurvey(updatedSurvey);
```
Comment (villejuutila) [inline src/components/screens/edit-surveys-screen.tsx:58]: Wrap this in try-catch

File: src/components/layout-components/survey-properties.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  ) => (
+    <WithDebounce
+      name={ name }
+      value={ value }
+      onChange={ onChange }
+      placeholder={ placeholder }
+      component={ props =>
+        <TextField
+          { ...props }
+          type="number"
```
Comment (villejuutila) [inline src/components/layout-components/survey-properties.tsx:66]: Are we certain that we want `type="number"`, same result could be applied with `pattern="[0-9]+"`? At least then there wouldn't be those annoying arrow buttons next to the input field 😆

File: src/components/layout-components/survey-properties.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+          fullWidth
+          placeholder={ strings.editSurveysScreen.editSurveyPanel.returnTimeout }
```
Comment (villejuutila) [inline src/components/layout-components/survey-properties.tsx:68]: Get this from props to make this a more generic function.

**Review by villejuutila — CHANGES_REQUESTED** [review]

File: src/components/layout-components/survey-properties.tsx (hunk 2 above)
Comment (EVP1989) [reply src/components/layout-components/survey-properties.tsx:66]: I think pattern might not be supported on every browser and those arrows can be removed with some css if needed.

**Review by villejuutila — APPROVED** [review]

**Pull request merged**